consoles/serial_screen.pm

```diff
@@ -26,10 +26,16 @@ use Carp 'croak';
 our $VERSION;
 
 sub new {
-    my ($class, $socket_fd) = @_;
+    my ($class, $fd_read, $fd_write) = @_;
     my $self = bless {class => $class}, $class;
-    $self->{socket_fd}    = $socket_fd;
+    $self->{fd_read}      = $fd_read;
+    $self->{fd_write}     = $fd_write // $fd_read;
     $self->{carry_buffer} = '';
+
+    my $old_fh = select($fd_write);
+    $| = 1;
+    select($old_fh);
```

Comment:
I don't think we need this?

Reply:
Your right from perldoc syswrite ` It bypasses buffered IO, so mixing this with reads (other than sysread))`

Going to remove it.